json-common/src/main/java/com.cerner.beadledom.json.common/model/ErrorDetail.java

```diff
@@ -0,0 +1,87 @@
+package com.cerner.beadledom.json.common.model;
```

Comment:
It looks like the package directory structure for the classes in this module got squished into a single directory with dots rather than a directory per package segment.

Reply:
Good catch! Made that fix: https://github.com/cerner/beadledom/pull/45/commits/ee2dd85c720353cbdc91bef9862ddd478bfcfcc8